### PR TITLE
fix hyperlinks to point to new repository url

### PR DESCRIPTION
**Pull request by andrei-ng**
- change references from `master` to `main` branch
 - change hyperlinks throught the repo to point to new repository url
 - similarly, fix the hyperlinks in the Rust book

### CODE REVIEW

**Comment by ndrezn** [comment]
Is this replacing #190?

**Comment by andrei-ng** [comment]
> Is this replacing #190?

No. It.is merely based on the same branch that #190 will merge. Once that is merged, I will rebase this. It involves a bit more work. All the examples and READMEs still use the old urls/refs.

File: examples/maps/src/main.rs [hunk 1]
```diff
@@ -10,12 +10,14 @@ fn scatter_mapbox() {
     let trace = ScatterMapbox::new(vec![45.5017], vec![-73.5673])
         .marker(Marker::new().size(25).opacity(0.9));
 
+    let token = "pk.eyJ1IjoiY2hyaWRkeXAiLCJhIjoiY2lxMnVvdm5iMDA4dnhsbTQ5aHJzcGs0MyJ9.X9o_rzNLNesDxdra4neC_A";
```
Comment (ndrezn) [inline examples/maps/src/main.rs:13]: Token seems to have snuck in

Comment (andrei-ng) [reply examples/maps/src/main.rs]: True. Well spotted!

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@
     <span> | </span>
     <a href="https://docs.rs/crate/plotly/">API Docs</a>
     <span> | </span>
-    <a href="https://github.com/igiagkiozis/plotly/blob/master/CHANGELOG.md">Changelog</a>
+    <a href="https://github.com/igiagkiozis/plotly/blob/main/CHANGELOG.md">Changelog</a>
```
Comment (ndrezn) [inline README.md:28]: Might also in this PR be worth changing the self-referential links to https://github.com/plotly/plotly.rs.

It looks like their are a bunch; same basic idea as changing the branch references.

**Comment by andrei-ng** [comment]
@ndrezn please review

**Review by ndrezn — APPROVED** [review]
💃 Awesome 🧼 PR. Will be great to get these new references published to cargo 🚀

**Pull request merged**